package.json

```diff
@@ -62,7 +62,7 @@
     "validate-commit-msg": "^2.14.0"
   },
   "peerDependencies": {
-    "eslint": "^3.12.0 || ^4 || ^5 || ^6 || ^7"
+    "eslint": "^3.12.0 || ^4 || ^5 || ^6 || ^7 || ^8.0.0-0"
```

Comment:
we're not going to release until eslint 8 is actually out, so we don't have to support prereleases forever.